tests/lib/snaps/test-snapd-userns/bin/sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+PS1='$ '
+exec /bin/sh "$@"
```

Comment:
missing end newline

Reply:
Thanks.

Reply:
Fixed in [fe71e56](https://github.com/snapcore/snapd/pull/12844/commits/fe71e567877b461269efe4f8f4a7d0f3bbdfb477)